data/README.rst

```diff
@@ -0,0 +1,107 @@
+Software Directory README
```

Comment:
Clarification: this is the very first time I've seen that extension being used. :)